### DISCUSSION TITLE
react-remove-scroll keeps give me warnings after use the Select in the Dialog component

### DISCUSSION THREAD

**Discussion by JhonM** [radix-ui/primitives#1594] [Help] [answered]
Hi,

When I use a `Select` component inside the `Dialog`, I get an error when clicking the `Select` trigger.
```
react-remove-scroll-bar: cannot calculate scrollbar size because it is removed (overflow:hidden on body 
    at RemoveScrollBar (http://localhost:3000/project-x/index.js:45320:29)
    at RemoveScrollSideCar (http://localhost:3000/project-x/index.js:45444:37)
```
I noticed that react-remove-scroll adds an overflow hidden on the `body` element that is causing this error. Is there any solution how to fix this?

For clarification I created a sandbox where I could reproduce it: https://codesandbox.io/s/select-in-dialog-primitive-react-remove-scroll-error-dfkobw

Thank You in advance and keep up the great work!

Jhon

**Comment by benoitgrelard** [comment, marked as answer]
Hi @JhonM, there are other issues/discussions about this.
Bottom line is: this is just a dev warning from the 3rd party library we use internally `react-remove-scroll`, you can safely ignore it or raise an issue with them.

**Reply by JhonM** [reply]
Thanks @benoitgrelard, I will ignore this.

**Reply by ArcanisCz** [reply]
Hi! Even if its just a warning, it pollutes console with false errors which is not nice. Also its just a rude from a library to (radix-ui and/or react-remove-scroll) to just write anything into console of app without ability to silence it. Any possibility it can be silenced on radix-ui level since its radix-ui using that library and not me (as a user of radix-ui)?

**Reply by angelod1as** [reply]
+1 to @ArcanisCz 
I'm trying to debug something, and this warning keeps polluting my console.

**Reply by andy-hook** [reply]
@ArcanisCz @angelod1as The most productive action to take here is to raise feedback with the library maintainer via [this open issue](https://github.com/theKashey/react-remove-scroll-bar/issues/35), or open a PR with the requested changes there.

**Reply by gunnartorfis** [reply]
Seems like they've released a fix @andy-hook 
https://github.com/theKashey/react-remove-scroll-bar/issues/35#issuecomment-1276809669
